components/list/ListItem.js

```diff
@@ -79,8 +81,15 @@ const factory = (ripple, ListItemLayout, ListItemContent) => {
       } = this.props;
       const children = this.groupChildren();
       const content = <ListItemLayout theme={theme} {...children} {...other} />;
+
       return (
-        <li className={`${theme.listItem} ${className}`} onClick={this.handleClick} onMouseDown={onMouseDown} onTouchStart={onTouchStart}>
+        <li
+          className={`${theme.listItem} ${className}`}
+          onClick={this.handleClick}
+          onMouseDown={onMouseDown}
+          onTouchStart={onTouchStart}
+        >
+          {altText ? <span className={theme.screenReader}>{altText}</span> : null}
```

Comment:
Looks good. I believe its convention to have the screen reader first read the text content within the element and then read the aria status or role after. Let's have the `altText` span as the last element within the `li`.